.buildkite/pipeline.yml

```
@@ -0,0 +1,22 @@
+steps:
+  - label: 'stack rebuild'
+    env:
+      STACK_ROOT: "/build/cardano-node.stack"
+    command:
+      - "rm -rf /build/cardano-node"
+      - "cp -R . /build/cardano-node"
+      - "cd /build/cardano-node"
+      - "nix-build scripts/buildkite -o stack-rebuild"
+      - "./stack-rebuild"
+    agents:
+      system: x86_64-linux
+
+  - label: 'nix-tools'
+    command: 'scripts/buildkite/check-nix-tools.sh'
```

```suggestion
    command: 'nix-build -A iohkLib.check-nix-tools -o check-nix-tools.sh && ./check-nix-tools.sh'
```
Update the iohk-nix revision and then you can delete `check-nix-tools.sh`.

---

My apologies, the above comment should have `iohkLib` replaced with `_lib`.